### PR TITLE
Advanced syntax highlighting

### PR DESCRIPTION
**Pull request by shendiaomo**
This is a follow-up to #2099 
The implementation uses https://github.com/alecthomas/chroma to highlight the inputs and convert it to a colored ASCII sequence that is written to stdout using `go-prompt`.
There're several pitfalls:
- strings across multiple lines are not highlighted correctly, because we can only get the current line from `go-prompt`
- the syntax is hard-coded to `MySQL`. I'll fix this later in some way
- because the highlighting is implemented in the bottom level, prompts in history searching mode are highlighted too

Everything else worked well. The result looks like:
image

### CODE REVIEW

File: cmd/repl/output.go [hunk 1]
```diff
@@ -25,35 +27,11 @@ type stdoutWriter struct {
...
-	"TABLES":    true,
-	"TO":        true,
-	"UPDATE":    true,
-	"USE":       true,
-	"USING":     true,
-	"VALUES":    true,
-	"WHERE":     true,
-	"WITH":      true,
-}
 var extendedKeywords = map[string]bool{
```
Comment (weiguoz) [inline cmd/repl/output.go:30]: Shall we highlight the `column`, `with` as the extended keywords?

Comment (shendiaomo) [reply cmd/repl/output.go:30]: I'm not sure哈. Unlike `TO`, `COLUMN` and `WITH` are used very often in standard SQL. Let's discuss this tomorrow.

Comment (brightcoder01) [reply cmd/repl/output.go:30]: I think that the highlight feature is for SQLFlow, more than MySQL. As a result, highlighting the keyword `TO`, `COLUMN` and `WITH` for SQLFlow would be more user-friendly.

Comment (shendiaomo) [reply cmd/repl/output.go:30]: Done.

**Review by weiguoz — APPROVED** [review]
LGTM

**Review by lhw362950217 — APPROVED** [review]
LGTM++

**Pull request merged**